packages/nameguard-sdk/package.json

```diff
@@ -39,6 +39,7 @@
   },
   "devDependencies": {
     "@types/node": "^20.6.2",
+    "@namehash/ens-utils": "workspace:*",
```

Comment:
Hmm.. I don't think it's right that this is listed only under `devDependencies`?

Reply:
Shouldn't it be under `dependencies` instead? Please confirm everything is configured correctly for us here and in the lockfile.